src/psbt.h

```diff
@@ -243,7 +246,7 @@ struct PSBTInput
                 }
                 // Unknown stuff
                 default:
-                    if (unknown.count(key) > 0) {
+                    if (!key_lookup.emplace(key).second) {
```

Comment:
I don't think this check for an unknown key is necessary in the same way the it is not needed for partial sigs or hd keypaths. It is a map of unknowns and we aren't checking if something is null.

Reply:
You are correct, I left it for visual consistency (probably should have done the same for `PSBT_IN_PARTIAL_SIG`).

If you feel strong about it, will revert it for the unknowns.

Reply:
Finally have reverted it for the unknowns.